python/PiFinder/ui/preview.py

```diff
@@ -113,31 +116,79 @@ def draw_reticle(self):
         )
 
         fov = 10.2
+        solve_pixel = self.shared_state.solve_pixel(screen_space=True)
         for circ_deg in [4, 2, 0.5]:
             circ_rad = ((circ_deg / fov) * 128) / 2
             bbox = [
-                64 - circ_rad,
-                64 - circ_rad,
-                64 + circ_rad,
-                64 + circ_rad,
+                solve_pixel[0] - circ_rad,
+                solve_pixel[1] - circ_rad,
+                solve_pixel[0] + circ_rad,
+                solve_pixel[1] + circ_rad,
             ]
             self.draw.arc(bbox, 20, 70, fill=self.colors.get(brightness))
             self.draw.arc(bbox, 110, 160, fill=self.colors.get(brightness))
             self.draw.arc(bbox, 200, 250, fill=self.colors.get(brightness))
             self.draw.arc(bbox, 290, 340, fill=self.colors.get(brightness))
 
+    def draw_star_selectors(self):
+        # Draw star selectors
+        if self.star_list.shape[0] > 3:
```

Comment:
what if only one star can be seen, why wouldn't we want to center on it?

Reply:
Good point, it's unlikely there would be one bright star without at least two others in the image, but no reason to limit it here.  I'll make that change.